lib/matplotlib/ticker.py

```diff
@@ -2908,9 +2941,9 @@ def __call__(self):
             vmin, vmax = vmax, vmin
 
         t0 = majorlocs[0]
-        tmin = round((vmin - t0) / minorstep)
-        tmax = round((vmax - t0) / minorstep) + 1
-        locs = (np.arange(tmin, tmax) * minorstep) + t0
+        tmin = ((vmin - t0) // minorstep + 1) * minorstep
+        tmax = ((vmax - t0) // minorstep + 1) * minorstep
+        locs = np.arange(tmin, tmax, minorstep) + t0
```

Comment:
This change seems unreleated, though I think it was included in a different recent PR

Reply:
Must have been an artifact when I set the branch up. I'll remove it